exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h

```diff
@@ -313,6 +315,50 @@ inline OtlpHeaders GetOtlpDefaultMetricHeaders()
   return result;
 }
 
+inline bool GetOtlpDefaultMetricsIsSslEnable()
+{
+  constexpr char kOtlpMetricsIsSslEnableEnv[] = "OTEL_EXPORTER_OTLP_METRICS_SSL_ENABLE";
+  constexpr char kOtlpIsSslEnableEnv[]        = "OTEL_EXPORTER_OTLP_SSL_ENABLE";
+
+  auto ssl_enable = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpMetricsIsSslEnableEnv);
+  if (ssl_enable.empty())
+  {
+    ssl_enable = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpIsSslEnableEnv);
+  }
+  if (ssl_enable == "True" || ssl_enable == "TRUE" || ssl_enable == "true" || ssl_enable == "1")
+  {
+    return true;
+  }
+  return false;
+}
+
+inline const std::string GetOtlpDefaultMetricsSslCertificatePath()
+{
+  constexpr char kOtlpMetricsSslCertificate[] = "OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE";
+  constexpr char kOtlpSslCertificate[]        = "OTEL_EXPORTER_OTLP_CERTIFICATE ";
+  auto ssl_cert_path =
+      opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpMetricsSslCertificate);
+  if (ssl_cert_path.empty())
+  {
+    ssl_cert_path = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpSslCertificate);
+  }
+  return ssl_cert_path.size() ? ssl_cert_path : "";
+}
+
+inline const std::string GetOtlpDefaultMetricsSslCertificateString()
+{
+  constexpr char kOtlpTracesSslCertificateString[] =
+      "OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE_STRING";
+  constexpr char kOtlpSslCertificateString[] = "OTEL_EXPORTER_OTLP_CERTIFICATE_STRING ";
```

Comment:
Is there a trailing space in this string liternal?

Reply:
Good catch, it is copy-paste error from traces. Had fixed for traces but forgot to do it here. Have fixed it now. Thanks.